test/e2e/quadlet/podmanargs.pod

```diff
@@ -0,0 +1,13 @@
+## assert-podman-pre-args "--foo"
+## assert-podman-pre-args "--bar"
+## assert-podman-pre-args "--also"
+## assert-podman-pre-args "--with-key=value"
+## assert-podman-pre-args "--with-space" "yes"
```

Comment:
Can we also check for the name?  It should be `systemd-podmanargs` if I am not mistaken.

Reply:
Same as for the`name.pod` case. This is already checked in the `basic.pod` case